static/js/popovers.js

```diff
@@ -206,6 +207,58 @@ exports.toggle_actions_popover = function (element, id) {
     }
 };
 
+function do_set_reminder(msgid, timestamp) {
+    var message = current_msg_list.get(msgid);
+    var link_to_msg = narrow.by_conversation_and_time_uri(message, true);
+    var command = compose.deferred_message_types.reminders.slash_command;
```

Comment:
Ugh, foo.bar.baz.yo is kind of long. :)  We may want to extract a `reminders.js` for some of this stuff, since a lot of this doesn't even touch the compose box.